PA_real_time_monitoring.md

```diff
@@ -77,53 +77,112 @@ const BIDDING_SLOW_WEIGHT = 0.1;
+There may be some errors that occur that are not visible in either `scoreAd()` or `generateBid()`. These include things like failures to fetch the bidding script, trusted real-time signals, or creative URL. In these cases, we allocate a certain portion of the resulting histogram for platform errors. While this can be done as a completely separate set of buckets that are immutable from JavaScript, we would still require the number of total contributions across all buckets to be capped at 1, so these platform-assisted contributions will come with a default weighting and participate in the prioritization algorithm just like regular contributions.
 
-The priority weight of platform contributions will be hardcoded and specified, so you can reason about the behavior across platform and developer contributions.
+The priority weight of platform contributions is hardcoded as 1, so you can reason about the behavior across platform and developer contributions.
 
+Platform contributions are reported in the report’s `platformHistogram` field, separately from regular contributions. See [below](#sending-reports-after-an-auction) for more details.
+
+| Bucket | Error |
+| --- | --- |
+| 0 | Bidding script fetch error |
+| 1 | Scoring script fetch error |
+| 2 | Trusted bidding signals fetch error |
+| 3 | Trusted scoring signals fetch error |
+
+Fetch errors include non-2xx response code, response header does not have “Ad-Auction-Allowed: true”, response body cannot be parsed as valid json for trusted signals, etc,.
 
 ## Sending reports after an auction
 
 After the auction completes, all opted-in participants (sellers, and buyers with interest groups present on device) will always emit one report per participant per auction, even if no ad wins or no `contributeToRealTimeHistogram()` calls are made. These reports will be sent to `domain.example/.well-known/interest-group/real-time-report` where `domain.example` would be replaced with the seller’s origin or the buyer’s interest group owner origin. These reports are not tied to specific interest groups, they are aggregated over all interest groups.The serialization format of the report is TBD, but see [below](#histogram-contributions-and-the-rappor-noise-algorithm) for the high level description of the encoding. Reporting domains may be configurable in the future. 
```

Comment:
>These reports will be sent to `domain.example/.well-known/interest-group/real-time-report` 

These are POSTed, yes?

Sending the low-entropy UACH for browser version would be useful.

Reply:
Yes, it is POSTed; we'll update the explainer to say this explicitly.

In terms of UACH, that's probably too much detail for this explainer.  It's probably worth noting more interest/motivation in UACH headers on other Protected Audience requests in #1031.

Reply:
Will do. Thank you for the reminder about that issue, @JensenPaul.

Reply:
Marking this comment thread as resolved as I added another mention of HTTP POST to the explainer.